### PR TITLE
fix(learn): Improve text for clarity

### PR DESCRIPTION
**Pull request by dipikabh**
### Description

Fixes the issue raised in https://github.com/orgs/mdn/discussions/659.
Also edits the next sentence for readability.

### CODE REVIEW

File: files/en-us/learn/accessibility/html/index.md [hunk 1]
```diff
@@ -254,7 +254,7 @@ Table layouts are a relic of the past — they made sense back when CSS support
 </footer>
 ```
 
-If you try our more modern structure example with a screen reader, you'll see that the layout markup no longer gets in the way and confuses the content readout. It is also much leaner and smaller in terms of code size, which means easier to maintain code, and less bandwidth for the user to download (particularly prevalent for those on slow connections).
+If you try our more modern structure example with a screen reader, you'll see that the layout markup no longer interferes with or causes confusion in the content readout. It is also much leaner and smaller in terms of code size, which means the code is easier to maintain and requires less bandwidth for users to download, making it particularly beneficial for those on slow connections.
```
Comment (estelle) [inline files/en-us/learn/accessibility/html/index.md:257]:
```suggestion
If you try our more modern structure example with a screen reader, you'll notice that the layout markup no longer interferes with or causes confusion in the content readout. It is also much leaner and smaller in terms of code size, which means the code is easier to maintain and requires less bandwidth for users to download, making it particularly beneficial for those on slow connections.
```

**Review by estelle — APPROVED** [review]
LGTM. I'm just suggesting a single word change since it may be heard, not seen. But fine as is too, so approving and letting you decide. :D

File: files/en-us/learn/accessibility/html/index.md (hunk 1 above)
Comment (dipikabh) [reply files/en-us/learn/accessibility/html/index.md]: Makes sense, thanks

**Comment by dipikabh** [comment]
Thanks for the review, @estelle!

**Pull request merged**